src/Page/Guide/View.elm

```diff
@@ -105,22 +105,36 @@ viewRow ( content1, content2, content3 ) =
         ]
 
 
-viewMaybeVideo : Maybe Page.Shared.Data.VideoMeta -> Html Msg
-viewMaybeVideo maybeVideoMeta =
+viewMaybeVideo : Language -> Maybe Page.Shared.Data.VideoMeta -> Html Msg
+viewMaybeVideo lang maybeVideoMeta =
+    let
+        t : Key -> String
+        t =
+            translate lang
+    in
     case maybeVideoMeta of
         Just aVideo ->
-            Page.Shared.View.viewVideo aVideo
+            div []
+                [ h2 [] [ viewHeaderIcon "../../../public/images/icons/NextDoorNature_icons-video.svg", text (t GuideVideoHeader) ]
```

Comment:
I think the image src path is wrong and could also be moved to a variable

Reply:
agreed